tests/validator20/validate_spec_test.py

```diff
@@ -423,3 +423,27 @@ def test_highlight_inconsistent_schema_object_validation(minimal_swagger_dict, s
     minimal_swagger_dict.update(swagger_dict_override)
     with pytest.raises(SwaggerValidationError):
         validate_spec(minimal_swagger_dict)
+
+
+def test_additional_properties_validated(minimal_swagger_dict, default_checks_spec_dict):
+    invalid_spec = {'type': 'object', 'required': ['foo']}
+    minimal_swagger_dict['definitions']['injected_definition'] = invalid_spec
```

Comment:
This line seems to have no effect (se line 432).

Reply:
ah yes my bad. already merged though